### ISSUE TITLE
CommonSpace App login problems

### ISSUE THREAD

**Issue by eozberk** [sidewalklabs/commonspace#27]
Two of my surveyors had issues creating an account and logging into the app on their iPhones:
- Password specifications were not identified when creating an account in the app, eliciting error window without explanation.
- Once logged in, assigned surveys did not appear in app.
- Password reset function does not appear to function properly.

My admin credentials worked for logging into the app, though it was not clear this was the case at first. If an email has been "whitelisted" to beta-test the admin, is the holder of that email address required to first set up account online via admin panel?

**Comment by mebreuer** [comment]
Hi @eozberk -- thanks for filing these! a couple questions to help us debug:
- can you elaborate a bit on the assigned surveys not appearing in app? did you confirm that the surveyor emails you entered in the admin panel match exactly the emails your surveyors used to sign up? and the studies are not marked as "completed"?
- which part of password reset function did not work? for example, did you receive the password reset email? did the link open to a valid webpage? 

thanks,
Matt

**Comment by eozberk** [comment]
Hi Matt,

Responses to your questions from my colleagues:

1. After logging into the app (I created a new account and password) I did not see a survey assigned to me. I was able to use the demo mode only. I also could not logout of the app. I repeatedly selected “Logout” and nothing happened. It seemed like the app froze. See attached screenshot when attempted again this morning.

2. I used the password reset option, it prompted me an e-mail was sent to reset my password. I reset my password, then tried signing in again, but it said invalid password (or something similar). I tried it again thinking I may have used the wrong password or hit a wrong key. I received the same invalid response. 

Hope that helps?

Erkin

![IMG_5531](https://user-images.githubusercontent.com/50636371/57798790-84f54680-771b-11e9-8cdc-3fc1b95eccfa.PNG)
